firmware/lib/Gantry/Axis.cpp

```diff
@@ -1,311 +1,569 @@
+/* **************************** Local Includes ***************************** */
 #include "Axis.h"
-#include "Debug.h"
-#include "Timer.h"
 #include "Movement.h"
+#include "Timer.h"
+
+/*****************************************************************************/
+/*                                  DEFINES                                  */
+/*****************************************************************************/
+
+/** TC IRQ number for x-axis velocity timer */
+#define IRQ_X_AXIS_VELOCITY    3
+/** TC IRQ number for x-axis acceleration timer */
+#define IRQ_X_AXIS_ACCEL       4
+
+/** TC IRQ number for y-axis velocity timer */
+#define IRQ_Y_AXIS_VELOCITY    0
+/** TC IRQ number for y-axis acceleration timer */
+#define IRQ_Y_AXIS_ACCEL       1
+
+/** Frequency of the slow clock */
+#define SCLK_FREQ              32768
+
+/** Minimum pulse duration to pass debouncing */
+#define DEBOUNCE_FILTER_MS     50
+
+/** Macro to determine if we've reached a target encoder count based on direction */
+#define REACHED_TARGET(_dir, _cur, _tar)             \
+    (((_dir) == DIR_POSITIVE && (_cur) >= (_tar)) || \
+    ((_dir) == DIR_NEGATIVE && (_cur) <= (_tar)))
+
+/** Maximum allowed velocity for an axis */
+#define VEL_MAX                25000
```

Comment:
Can we add its units in comment or in the variable name for clarity. In this case, it looks like counts?

Reply:
It's actually steps / second here, so evidently good point. I actually realize I need to move this up a couple levels, so will do that and add units in a comment